pkg/cmd/cli/describe/describer.go

```diff
@@ -1322,3 +1305,47 @@ func (d *ClusterRoleBindingDescriber) Describe(namespace, name string) (string,
 	role, err := d.ClusterRoles().Get(roleBinding.RoleRef.Name)
 	return DescribeRoleBinding(authorizationapi.ToRoleBinding(roleBinding), authorizationapi.ToRole(role), err)
 }
+
+func describeBuildTriggerCauses(causes []buildapi.BuildTriggerCause, out *tabwriter.Writer) {
+	fmt.Fprintf(out, "\nTrigger causes:")
+	if causes == nil {
+		fmt.Fprintf(out, " N/A")
```

Comment:
i debated the same suggestion, but i don't like the absence of information.  the presence of an "unknown" value is a clue to the user that there could be a value there and perhaps they should figure out how to make that value appear.